update-analyzer/action.yml

```
@@ -9,12 +9,15 @@ inputs:
   ticket:
     description: 'The Jira ticket number (e.g., SONAR-12345 or SC-12345). This determines the target product.'
     required: true
-  plugin-language:
+  plugin_language:
     description: 'The language key of the plugin to update (e.g., architecture, java, csharp).'
     required: true
-  github-token:
-    description: 'A GitHub token with permissions to create pull requests in the target repository.'
+  secret_name:
+    description: 'Name of the secret to fetch from the vault that has access to the target repository.'
```

```suggestion
    description: 'Name of the secret to fetch from the vault that has access to create pull requests in the target Github repository.'
```
IMO the current description sounds too generic

---

Good catch, thanks. Will combine it with Teemu's comment and make it explicit that it's fetching a github token from the secret.